src/ripple/protocol/STObject.h

```diff
@@ -538,13 +538,22 @@ class STObject
     }
 
 private:
-    void add (Serializer & s, bool withSigningFields) const;
+    enum WhichFields : bool
```

Comment:
Glad you're using more-of named-parameters than just unnamed bools. This related back to your comment in a previous PR. 👍